apps/kyc-server/src/components/dashboard/table/user-actions-menu.tsx

```
@@ -0,0 +1,84 @@
+import { Button } from '~/components/base/button'
+import {
+	DropdownMenu,
+	DropdownMenuContent,
+	DropdownMenuItem,
+	DropdownMenuSeparator,
+	DropdownMenuTrigger,
+} from '~/components/base/dropdown-menu'
+import { useKycActions } from '~/hooks/use-kyc-actions'
+
+interface UserActionsMenuProps {
+	user: any // TODO: should probably use proper types but this works for now
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)